### DISCUSSION TITLE
I managed to get things working with python 3.11, bare metal linux...

### DISCUSSION THREAD

**Discussion by ilikenwf** [vladmandic/sdnext#783] [General]
I'm using a 1080ti 11gb overclocked, and while my settings vary somewhat with cuda, I notice that even when using full precision, while also using token merging, that I range 4-5.25, sometimes higher, iterations per second on 512x512 where I once was more around 3.75-4.20. 

It could be versions of things or it could be that I rebuilt my venv to do it...just food for thought. I'm also running the master branch version of xformers and the repositories subdirs. For face detection to use the master branch it's just a 2-4 line change in the code.

**Comment by vladmandic** [comment]
python 3.11 doesn't make that much of a difference for sure, its other things.
i'm not supporting 3.11 because its not supported by several packages, notably some upscalers and background removal don't work.
but otherwise, its "fine" if you want to experiment with it (thats what `--experimental' flag is for), just don't open any issues while you're on 3.11 unless you're willing to contribute :)

**Comment by ilikenwf** [comment]
Makes sense - I haven't used that flag, but one thing was removing the BG remover plugin from my test branch...

I may at least try to contribute a fix for the "current" face detection/correction stuff sometime, since the only breaking change is something that got renamed from face_det to face_detector if memory serves...

I'm impressed my used $250 1080ti is holding it's own so well.

If it would help to have versions of any of my packages or anything, I'd be happy to share. I have my venv setup to use system packages where possible, as well, because I'm too lazy to build torch and it's in the Arch repos.
